README.md

```diff
@@ -11,7 +11,10 @@ We’re releasing MetaVoice-1B under the Apache 2.0 license, *it can be used wit
 
 Try out the [demo](https://ttsdemo.themetavoice.xyz/)!
 
-## Installation
+## Installation  
+
+**Pre-requisites:** Python 3.10, GPU with ≥24GB RAM.
```

Comment:
nit Think 3.10 isn't a hard requirement. 3.11 works for sure...

Reply:
okay, ill keep 3.10 & 3.11 for now